### PR TITLE
qa: use exisitng ignorelist override list for fs:mirror[-ha]

### PR DESCRIPTION
**Pull request by vshankar**
The custom ignorelist yaml would also require `POOL_APP_NOT_ENABLED` to be ignored. The custom yaml is unnecessary anyway.

Fixes: http://tracker.ceph.com/issues/62482

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: qa/suites/fs/mirror-ha/overrides/whitelist_health.yaml [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-      - \(FS_DEGRADED\)
-      - \(MDS_FAILED\)
-      - \(MDS_DEGRADED\)
-      - \(FS_WITH_FAILED_MDS\)
-      - \(MDS_DAMAGE\)
-      - \(MDS_ALL_DOWN\)
-      - \(MDS_UP_LESS_THAN_MAX\)
-      - \(FS_INLINE_DATA_DEPRECATED\)
-      - Reduced data availability
-      - Degraded data redundancy
```
Comment (joscollin) [inline qa/suites/fs/mirror-ha/overrides/whitelist_health.yaml:14]: What about these two items, which are not in `qa/cephfs/overrides/ignorelist_health.yaml` ?

**Review by joscollin — COMMENTED** [review]
Do the same for `qa/suites/fs/valgrind/mirror/overrides/whitelist_health.yaml` too?

File: qa/suites/fs/mirror-ha/overrides/whitelist_health.yaml (hunk 1 above)
Comment (vshankar) [reply qa/suites/fs/mirror-ha/overrides/whitelist_health.yaml:14]: I don't see these warnings in my latest run, so they are likely unnecessary.

**Review by batrick — APPROVED** [review]

**Comment by batrick** [comment]
https://pulpito.ceph.com/pdonnell-2023-09-12_14:07:50-fs-wip-batrick-testing-20230912.122437-distro-default-smithi/7395044

**Comment by vshankar** [comment]
> https://pulpito.ceph.com/pdonnell-2023-09-12_14:07:50-fs-wip-batrick-testing-20230912.122437-distro-default-smithi/7395044

Good catch.

**Comment by vshankar** [comment]
@joscollin fixed and updated.

**Review by joscollin — APPROVED** [review]

**Comment by vshankar** [comment]
https://tracker.ceph.com/projects/cephfs/wiki/Main#20-Sep-2023

**Comment by vshankar** [comment]
@pdvian https://tracker.ceph.com/issues/62482#note-13

**Pull request merged**